### PR TITLE
feat: creates useUpdateProfileDetails w/ example

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import {
+  ProfileFieldsFragment,
+  useActiveProfile,
+  useUpdateProfileDetails,
+} from '@lens-protocol/react';
+import { useForm } from 'react-hook-form';
```
[the remark this answers is not in the document]
Comment (reecejohnson) [reply examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx:6]: by "raw `FormData`" , i mean: https://developer.mozilla.org/en-US/docs/Web/API/FormData

File: examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+      <label>
+        Location:
+        <br />
+        <input
+          required
+          type="text"
+          placeholder="Where are you?"
+          disabled={isPending}
+          defaultValue={profile.attributes.location.toString()}
```
Comment (reecejohnson) [inline examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx:70]: I'm seeing an error thrown on this line when running locally with my profile `reecej.test`

Seems I don't have any attributes set

image

File: examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cesarenaldi) [reply examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx:6]: Good call. Will merged all these 4 PRs into one integration branch and simplify the example.

File: examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx (hunk 2 above)
Comment (cesarenaldi) [reply examples/web-wagmi/src/profiles/UseUpdateProfileDetails.tsx:70]: could it be a not-recompiled `api-bindings`?

**Pull request merged**